airflow_provider_hightouch/hooks/hightouch.py

```diff
@@ -61,41 +85,62 @@ def poll(
                 )
                 continue
 
-            # TODO: Handle all errors
-            if state in (self.RUNNING, self.PENDING):
+            if state in (self.CANCELLED, self.FAILED, self.ABORTED):
+                raise AirflowException(
+                    f"Job {sync_id} failed to complete with stauts {state}"
```

Comment:
I like to read this new format strings as `F Job <id> failed to complete with status` (btw, misspelled status)